travis_v3io.yaml

```diff
@@ -0,0 +1,5 @@
+container: "bigdata"
+workers: 1
+webApiEndpoint: "webapi.iguazio.app.mianvbzjrnzf.dev.trl.iguazio.com:8081"
+username: "iguazio@Default"
+password: "usUvWsWomVveV7Lc"
```

Comment:
Why is this here? https://docs.travis-ci.com/user/pull-requests#pull-requests-and-security-restrictions